pf/sound.py

```diff
@@ -1,21 +1,57 @@
+from .features import FEATURES,NON_DIACRITIC_TO_FEATURES
+from .ipa import *
 class Sound(object):
-    def __init__(self, content: str, type_: str, features: list):
+    def __init__(self, content: str, type_: str= None, features: dict=None):
             self.content = content
-            self.features = features
-            self.type = type_
+            self.features = self.translate_features()
+            self.type_ = type_
 						
     def __repr__(self, singular=False):
         output = self.content + " "
         if singular:
-            for feature in self.features:
+            for feature in self.features.values():
                 output += feature + " "
         return output[:-1]
 
     def __eq__(self, other):
         return self.content == other.content
 
     def compare(self, other):
```

Comment:
Consider renaming to something more descriptive, like `common_features(self, other)`